ui/DesignSystem/Component/SourceBrowser/RevisionSelector.svelte

```diff
@@ -162,14 +174,26 @@
   <div class="revision-dropdown" hidden={!expanded}>
     {#each revisions as repo}
       <div class="peer">
-        <Avatar
-          avatarFallback={repo.identity.avatarFallback}
-          style="display: flex; justify-content: flex-start; margin-right: 8px;"
-          size="small"
-          variant="circle" />
-        <p class="typo-text-bold typo-overflow-ellipses">
-          {repo.identity.metadata.handle || repo.identity.shareableEntityIdentifier}
-        </p>
+        <div style="display: flex;">
+          <Avatar
+            avatarFallback={repo.identity.avatarFallback}
+            style="display: flex; justify-content: flex-start; margin-right:
+            8px;"
+            size="small"
+            variant="circle" />
+          <p class="typo-text-bold typo-overflow-ellipses">
+            {repo.identity.metadata.handle || repo.identity.shareableEntityIdentifier}
+          </p>
+        </div>
+        <Tooltip value="Go to profile" position="top">
+          <div
+            class="open-profile"
+            on:click={() => {
+              repo.identity.peerId === session.identity.peerId ? push(path.profileProjects()) : push(path.userProfile(repo.identity.peerId));
```

Comment:
Inlining this gets a bit unwieldy. The least it would help have this up in the script block.

Reply:
https://github.com/radicle-dev/radicle-upstream/pull/795/commits/b23b5b3135dd757c5fe4f566058dc012e401b910